<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormSetting</class>
 <widget class="QWidget" name="FormSetting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>745</width>
    <height>479</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel, QSlider{
	background-color: rgba(0,0,0,0)
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="1" column="0">
    <widget class="QWidget" name="widget_CMD" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>600</width>
       <height>60</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>70</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">
#widget_CMD:hover{
	background-color: rgba(255,255,255,40);
	/*background-color: transparent;*/
	padding: 10px;
	border-left: 3px solid rgb(130, 130, 130);
}</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="1">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="2">
       <widget class="SwitchButton" name="switchButton_cmd" native="true">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_3">
        <property name="minimumSize">
         <size>
          <width>53</width>
          <height>42</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>幼圆</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>开启命令行</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QWidget" name="widget_Opacity" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>600</width>
       <height>60</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>70</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">
#widget_Opacity:hover{
	background-color: rgba(255,255,255,40);
	/*background-color: transparent;*/
	padding: 10px;
	border-left: 3px solid rgb(130, 130, 130);
}</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="minimumSize">
         <size>
          <width>53</width>
          <height>42</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>幼圆</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>界面透明度</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSlider" name="horizontalSlider">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="SwitchButton" name="switchButton_opacity" native="true">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QWidget" name="widget_DeleteAll" native="true">
     <property name="minimumSize">
      <size>
       <width>600</width>
       <height>60</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>70</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">#widget_DeleteAll:hover{
	background-color: rgba(255,255,255,40);
	/*background-color: transparent;*/
	padding: 10px;
	border-left: 3px solid rgb(130, 130, 130);
}</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_6">
      <item row="0" column="0">
       <widget class="QLabel" name="label_5">
        <property name="font">
         <font>
          <family>幼圆</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>删除所有数据</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QToolButton" name="toolButton_DeleteAll">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Microsoft YaHei UI Light</family>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QToolButton{
	background-color: #2c3e50;
	border: 4px solid #2c3e5080;
    border-radius: 5px;
}

/* 鼠标悬停 */
QToolButton:hover{
	/*background-color: rgba(236, 240, 241, 0.5);*/
	border: 1px solid  rgba(255, 0, 0, 100);
	color: red;
	font-weight: bold;
}

/* 点击和按下 */
QToolButton:pressed,QToolButton:checked{
	border-left: 2px outset #34495e80;  /* 左边框宽度3像素、突出显示、颜色为深深灰 */
	border-top: 2px outset #34495e80;
	border-right: none;
	border-bottom:none;
	background-color: #34495e
}
</string>
        </property>
        <property name="text">
         <string>DELETE ALL</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>500</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QWidget" name="widget_Reload" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>600</width>
       <height>60</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>70</height>
      </size>
     </property>
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">
#widget_Reload:hover{
	background-color: rgba(255,255,255,40);
	/*background-color: transparent;*/
	padding: 10px;
	border-left: 3px solid rgb(130, 130, 130);
}</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <family>幼圆</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>重载数据</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="2">
       <widget class="QToolButton" name="toolButton_Reload">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Microsoft YaHei UI Light</family>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QToolButton{
	background-color: #2c3e50;
	border: 4px solid #2c3e5080;
    border-radius: 5px;
}

/* 鼠标悬停 */
QToolButton:hover{
	/*background-color: rgba(236, 240, 241, 0.5);*/
	border: 1px solid  #bdc3c7;
	color: #ecf0f1;
	font-weight: bold;
}

/* 点击和按下 */
QToolButton:pressed,QToolButton:checked{
	border-left: 2px outset #34495e80;  /* 左边框宽度3像素、突出显示、颜色为深深灰 */
	border-top: 2px outset #34495e80;
	border-right: none;
	border-bottom:none;
	background-color: #34495e
}
</string>
        </property>
        <property name="text">
         <string>RELOAD</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="5" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0" alignment="Qt::AlignTop">
    <widget class="QWidget" name="widget_Save" native="true">
     <property name="minimumSize">
      <size>
       <width>600</width>
       <height>60</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>70</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">#widget_Save:hover{
	background-color: rgba(255,255,255,40);
	/*background-color: transparent;*/
	padding: 10px;
	border-left: 3px solid rgb(130, 130, 130);
}</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="2">
       <widget class="QToolButton" name="toolButton_Save">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Microsoft YaHei UI Light</family>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QToolButton{
	background-color: #2c3e50;
	border: 4px solid #2c3e5080;
    border-radius: 5px;
}

/* 鼠标悬停 */
QToolButton:hover{
	/*background-color: rgba(236, 240, 241, 0.5);*/
	border: 1px solid  #bdc3c7;
	color: #ecf0f1;
	font-weight: bold;
}

/* 点击和按下 */
QToolButton:pressed,QToolButton:checked{
	border-left: 2px outset #34495e80;  /* 左边框宽度3像素、突出显示、颜色为深深灰 */
	border-top: 2px outset #34495e80;
	border-right: none;
	border-bottom:none;
	background-color: #34495e
}
</string>
        </property>
        <property name="text">
         <string>SAVE</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_4">
        <property name="font">
         <font>
          <family>幼圆</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>保存数据</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SwitchButton</class>
   <extends>QWidget</extends>
   <header>switchbutton.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
